Dockerfile

```diff
@@ -3,27 +3,33 @@ ARG CUDA_VERSION="11.8.0"
 FROM mambaorg/micromamba:1.5.1 as micromamba
 FROM ${REGISTRY}/nvidia/cuda:${CUDA_VERSION}-devel-ubuntu22.04 as base
 
-#####
-# Setup user & common tools
-#####
-RUN apt update \
-  && apt install -y git ninja-build \
-  && rm -rf /var/lib/apt/lists/*
-
 #####
 # Setup micromamba
 #####
 
 USER root
 
 ARG MAMBA_USER=yi
-ARG MAMBA_USER_ID=56789
-ARG MAMBA_USER_GID=56789
+ARG MAMBA_USER_ID=1000
+ARG MAMBA_USER_GID=100
```

Comment:
Are there any functional changes here, or is it to be more consistent with convention?

Reply:
I would say it's more of a convention.

See https://askubuntu.com/a/991318